manifests/repo.pp

```diff
@@ -77,8 +77,13 @@
         $_repo_location = $repo_location
         $description = 'MongoDB Custom Repository'
       } else {
-        $_repo_location = "https://repo.mongodb.org/zypper/suse/\$releasever_major/mongodb-org/${version}/\$basearch/"
-        $description = 'MongoDB Repository'
+        if $use_enterprise_repo {
+          $_repo_location = "https://repo.mongodb.com/zypper/suse/\$releasever_major/mongodb-enterprise/${version}/\$basearch/"
+          $description = 'MongoDB Enterprise Repository'
+        } else {
+          $_repo_location = "https://repo.mongodb.org/zypper/suse/\$releasever_major/mongodb-org/${version}/\$basearch/"
+          $description = 'MongoDB Repository'
+        }
```

Comment:
We don't have tesing on Suse but the enterprise repo is there so I guess it's ok to add it.